package.json

```diff
@@ -28,6 +28,10 @@
     "url": "https://github.com/seegno/jscs-config-seegno/issues"
   },
   "homepage": "https://github.com/seegno/jscs-config-seegno#readme",
+  "dependencies": {
+    "flora-sql-parser": "^0.2.3",
+    "lodash.get": "^3.7.0"
```

Comment:
I thought a bit about this and I think it is worth to keep the dependency, otherwise the `if` clauses (particularly the one for `VariableDeclarations` will have a lot of conditions to check for existence and stuff like that.